src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -86,7 +88,7 @@ set_target_properties(System.Globalization.Native-Static PROPERTIES OUTPUT_NAME
 
 install (TARGETS System.Globalization.Native-Static DESTINATION .)
 
-if(NOT CLR_CMAKE_TARGET_OSX AND NOT CLR_CMAKE_TARGET_IOS)
+if(NOT CLR_CMAKE_TARGET_OSX AND NOT CLR_CMAKE_TARGET_IOS AND NOT CLR_CMAKE_TARGET_ANDROID)
```

Comment:
Can this be simplified to `if(CLR_CMAKE_TARGET_LINUX AND NOT CLR_CMAKE_TARGET_ANDROID)` ?

Reply:
If we wanted to keep the same behavior as before we'd need to add FreeBSD/NetBSD/OpenBSD too so it'd be much longer :)

The best option would be to skip this check if we don't have `ldd` which is what the `verify-so.sh` script is using.

Reply:
Ah right. We can leave as is if we don't want to complicate this PR.